_posts/2019-06-16-good-first-issue.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "goodfirstissue - what, why and how?"
+description: In this post Rajat will share what triggered him to write goodfirstissue app, where was it deployed and how it was a nobrainer to move to OpenFaaS cloud.
+date: 2019-06-16
+image: /images/good-first-issue/anukrati-omar-1206574-unsplash.jpg
+categories:
+  - templates
```

Comment:
Is this using `templates`?

How about adding `gitops` and `bot` maybe also `end-user`?